DataCollection/src/Migrations/Configuration.cs

```diff
@@ -13,9 +16,42 @@ public Configuration()
             SetSqlGenerator("System.Data.SqlClient", new CustomSqlServerMigrationSqlGenerator());
         }
 
-        protected override void Seed(FlightNode.DataCollection.Infrastructure.Persistence.DataCollectionContext context)
+        protected override void Seed(DataCollectionContext context)
+        {
+            LoadDisturbanceTypesForWaterbirdForaging(context);
+            LoadTidesForWaterbirdForaging(context);
+            LoadWeatherForWaterbirdForaging(context);
```

Comment:
This is the initial data load for the three helper tables that were added: Weather, Tides, and DisturbanceType.